package.json

```diff
@@ -35,7 +35,7 @@
     "forever-agent": "~0.6.1",
     "form-data": "~2.3.1",
     "har-validator": "~5.0.3",
-    "hawk": "~6.0.2",
+    "hawk": "~7.0.0",
```

Comment:
Did you check that this is sufficient and doesn't break anything? Hawk did a major version upgrade for a reason. The api is now not callback based anymore [0].